README.md

```diff
@@ -0,0 +1,10 @@
+# selenium-ta-project
+
+To run tests execute following command:
+
+```
+mvn -Dbrowser=chrome test
```

Comment:
not able to run the suite using this command

Reply:
fixed